### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by saugat135**

### CODE REVIEW

File: test/index.html [hunk 1]
```diff
@@ -645,6 +645,6 @@ <h2 class="heading2" style="text-align:justify">Ask a question</h2><br><br>
 * Copyright (c) 2012 Karl Swedberg; Licensed MIT, GPL */
 </script>
 <!--Smooth Scroller Ends Here-->
-
+<div>This is the changes</div>
```
Comment (vsujan) [inline test/index.html]: just a test comment

Comment (laxmankhanal) [inline test/index.html]: maintain proper standards

File: test/index.html [hunk 2]
```diff
@@ -645,6 +645,6 @@ <h2 class="heading2" style="text-align:justify">Ask a question</h2><br><br>
 * Copyright (c) 2012 Karl Swedberg; Licensed MIT, GPL */
 </script>
 <!--Smooth Scroller Ends Here-->
```
Comment (laxmankhanal) [inline test/index.html:647]: check

File: test/index.html (hunk 1 above)
Comment (sanjaymhj) [inline test/index.html]: second test

File: test/index.html (hunk 2 above)
Comment (saugat135) [inline test/index.html:647]: :ok:

File: test/index.html (hunk 1 above)
Comment (saugat135) [inline test/index.html]: :ok:

**Comment by laxmankhanal** [comment]
:+1:

**Comment by sanjaymhj** [comment]
:+1:

**Pull request merged**